indexer/FastIndexer.dockerfile

```
@@ -0,0 +1,47 @@
+FROM rust:1.79 AS builder
+WORKDIR /tmp/indexer
```

```suggestion
FROM rust:1.79.0-bookworm AS builder
WORKDIR /tmp/indexer
```

This will make sure the builder doesn't break since it's using bookworm. You could also elect to use bookworm-slim